docs/en/user_guides/4_train_test.md

```diff
@@ -218,3 +218,92 @@ You can check [the source code](../../../tools/slurm_test.sh) to review full arg
 CUDA_VISIBLE_DEVICES=0,1,2,3 GPUS=4 MASTER_PORT=29500 sh tools/slurm_train.sh ${PARTITION} ${JOB_NAME} config1.py ${WORK_DIR}
 CUDA_VISIBLE_DEVICES=4,5,6,7 GPUS=4 MASTER_PORT=29501 sh tools/slurm_train.sh ${PARTITION} ${JOB_NAME} config2.py ${WORK_DIR}
 ```
+
+## Testing and saving predicted segment files
+
+### Basic Usage
+
+When you want to save the results, you can specify the `output_dir` in     `test_evaluator`.
+
+```shell
+python tools/test.py ${CONFIG_FILE} ${CHECKPOINT_FILE} --cfg-options test_evaluator.output_dir=${OUTPUT_DIR}
+```
+
+Here is an example to save the predicted results from model `fcn_r50-d8_4xb4-80k_ade20k-512x512` on ADE20k validatation dataset.
+
+```shell
+python tools/test.py configs/fcn/fcn_r50-d8_4xb4-80k_ade20k-512x512.py ckpt/fcn_r50-d8_512x512_80k_ade20k_20200614_144016-f8ac5082.pth --cfg-options test_evaluator.output_dir=work_dirs/format_results
+```
```

Comment:
Should we add an argument to the `test.py` script like mmyolo?
https://github.com/open-mmlab/mmyolo/blob/main/tools/test.py#L28